src/Plugin.Maui.Audio/AudioPlayer.shared.cs

```diff
@@ -15,4 +16,36 @@ public void Dispose()
 
         GC.SuppressFinalize(this);
     }
+
+#if (NET6_0 && !ANDROID && !IOS && !MACCATALYST && !WINDOWS)
```

Comment:
I think before we explored having a specific naming convention for a file; something like `AudioPlayer.net.cs`. I am never really a fan of compiler directives but this is quite possibly clearer than the conditionally compiled file

Reply:
I was thinking similarly. It seems like a net6.0 target is something of a grey area - not accounted for in https://docs.microsoft.com/en-us/dotnet/maui/platform-integration/configure-multi-targeting